src/components/RecordTypesSelect/components/RecordItem.js

```diff
@@ -56,13 +58,21 @@ export const RecordItem = memo(({
     />
   );
 
+  // TODO: Disabling options should be removed after implentation is done
+  const isOptionDisabled = FOLIO_RECORD_TYPES_TO_DISABLE.some(option => option === item.type);
+
   const initialButton = (
     <div // eslint-disable-line jsx-a11y/click-events-have-key-events
       data-test-record-item
       tabIndex="0"
       role="button"
       id={incomingRecord.type}
-      className={classNames(css.item, { [css.clickableItem]: isEditable }, className)}
+      className={classNames(
+        css.item,
+        { [css.clickableItem]: isEditable },
+        className,
+        isOptionDisabled && css.disabledItem,
```

Comment:
Consider adding a `disabled` field to `itemMeta` in the `<RecordSelect>` `recordsData` const instead of creating a `FOLIO_RECORD_TYPES_TO_DISABLE` const because `recordsData` is a config, so I think it makes sense to store such property there.

Reply:
As for me it's more convenient and logical to save all disabled options in one place, and use them in all profiles, where these options should be selected.